src/pages/index.js

```diff
@@ -170,14 +297,23 @@ cardModalCloseBtn.addEventListener("click", () => {
   closeModal(cardModal);
 });
 
-editFormElement.addEventListener("submit", handleEditFormSubmit);
-cardForm.addEventListener("submit", handleAddCardSubmit);
+avatarModalBtn.addEventListener("click", () => {
+  openModal(avatarModal);
+  console.log("test");
+});
 
-initialCards.forEach((item) => {
-  const cardElement = getCardElement(item);
-  cardsList.append(cardElement);
+avatarModalCloseBtn.addEventListener("click", () => {
+  closeModal(avatarModal);
+  console.log("test");
 });
 
+avatarForm.addEventListener("submit", handleAvatarSubmit);
+
+deleteForm.addEventListener("submit", handleDeleteSubmit);
+
+editFormElement.addEventListener("submit", handleEditFormSubmit);
+cardForm.addEventListener("submit", handleAddCardSubmit);
+
 const newPostBtn = document.querySelector("#post-btn");
 newPostBtn.addEventListener("click", () => {
   const buttonElement = cardForm.querySelector(".modal__submit-btn");
```

Comment:
Please, delete it

You need to `disable` the button only after **resetting** the form when you have submitted a card otherwise users may close the modal by accident and they don’t expect that the button will be disabled on opening if the inputs have **valid** data which they haven’t submitted yet.

You need to **disable** the button only in a `then` block after a **successful response** to be able to click it again if there is a server error.

Reply:
Please, add ` disableButton(submitButton, config.inactiveButtonClass);` in line 251 to disable the card save button after submission